src/screens/FreezeFunds/04-ValidationSuccess.js

```diff
@@ -39,7 +35,7 @@ const useTimer = (timer: number) => {
 };
 
 type Props = {
-  account: ?(TokenAccount | Account),
+  account: ?Account,
```

Comment:
I think this should either be `AccountLike` OR you should drop parentAccount concept ?